src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+                    ),
+                    None,
+                )
+                if tempEntry_domain is not None:
+                    if debug_on:
+                        logger.info(
+                            f"{termColors.WARNING} DUPLICATE Verisign entries found for domain: {new_entry_domainName} {termColors.ENDC}"  # noqa
+                        )
+                    if new_entry_domainName not in duplicate_domains:
+                        duplicate_domains.append(new_entry_domainName)
+                if tempEntry_domainUserPair is not None:
+                    if debug_on:
+                        logger.info(
+                            f"""{termColors.WARNING} DUPLICATE Verisign entries found for domain - user {termColors.BackgroundLightYellow} PAIR {termColors.ENDC}{termColors.WARNING}:  
+                            {new_entry_domainName} - {new_entry_email} {termColors.ENDC}"""  # noqa
+                        )
+                    if tempEntry_domainUserPair not in duplicate_domain_user_combos:
+                        duplicate_domain_user_combos.append(tempEntry_domainUserPair)
+                else:
+                    try:
+                        existingEntry = TransitionDomain.objects.get(
+                            username=new_entry_email, domain_name=new_entry_domainName
+                        )
+
+                        if existingEntry.status != new_entry_status:
+                            # DEBUG:
+                            if debug_on:
+                                logger.info(
+                                    f"""{termColors.OKCYAN}
+    Updating entry: {existingEntry}
+    Status: {existingEntry.status} > {new_entry_status}
+    Email Sent: {existingEntry.email_sent} > {new_entry_emailSent}
+    {termColors.ENDC}"""
+                                )
+
+                            existingEntry.status = new_entry_status
+
+                        existingEntry.email_sent = new_entry_emailSent
+                        existingEntry.save()
+                    except TransitionDomain.DoesNotExist:
+                        # no matching entry, make one
+                        newEntry = TransitionDomain(
+                            username=new_entry_email,
+                            domain_name=new_entry_domainName,
+                            status=new_entry_status,
+                            email_sent=new_entry_emailSent,
+                        )
+                        to_create.append(newEntry)
+                        total_new_entries += 1
+
+                        # DEBUG:
+                        if debug_on:
+                            logger.info(
+                                f"{termColors.OKCYAN} Adding entry {total_new_entries}: {newEntry} {termColors.ENDC}"  # noqa
+                            )
+                    except TransitionDomain.MultipleObjectsReturned:
+                        logger.info(
+                            f"""
+{termColors.FAIL}
```

Comment:
**suggestion** another nitpick can you run black on this file? this indent is so wonky

Reply:
I did run black on this.  I also don't like the indent *however*, it is a trade-off where the terminal output follows exactly the indentation presented in the string.  In order to avoid a messy terminal output (due to the 88char limit from the linter), back-indenting was the road I chose.  Not super happy with it myself.  Honestly I think we need to increase the 88char limit.  There is a point where the linter's intent to enforce clean code actually harms readibility.

Reply:
+1 regarding the 88 char limit - I think it needs to be increased slightly personally